Pages/Admin/Locations/Edit.cshtml.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using ChemStoreWebApp.Models;
+
+namespace ChemStoreWebApp.Pages.Admin.Location
+{
+    public class EditModel : PageModel
```

Comment:
Add authorization attributes to these page classes. See the team StackOverflow for the syntax.

Reply:
Done in latest